com.wiiznokes.fan-control.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "com.wiiznokes.fan-control",
```

Comment:
the domain `wiiznokes.com` doesn't exist. therefor the app id is invalid, despite you claim the domain is controlled by the upstream developers.

Reply:
> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
Ho mb, what should i use instead ? I don't have a website for this project, and don't plan to add one for now

Reply:
does `io.github.wiiznokes.fan-control` would work ?

Reply:
changed to `io.github.wiiznokes.fan-control`